packages/gcc7.rb

```diff
@@ -0,0 +1,37 @@
+require 'package'
+
+class Gcc7 < Package
+  description 'The GNU Compiler Collection includes front ends for C, C++, Objective-C, Fortran, Ada, and Go.'
+  homepage 'https://www.gnu.org/software/gcc/'
+  version '7.3.0'
+
+  source_url 'https://github.com/gcc-mirror/gcc/archive/gcc-7_3_0-release.zip'
+
+  source_sha256 'd8cb199f0d98bd83e3dac645126722e44137c24cbe5e87e1113dd4270f7530c6'
+
+  depends_on 'buildessential'
+  depends_on 'binutils'
+  depends_on 'gmp'
+  depends_on 'mpfr'
+  depends_on 'mpc'
+  depends_on 'isl'
+  depends_on 'cloog'
+  depends_on 'glibc'
+  depends_on 'gawk'
+
+  def self.build
+    system "mkdir", "build"
+    Dir.chdir "build" do
+      ENV["LIBRARY_PATH"] = "/usr/local/lib64"
```

Comment:
This should definitely be `ENV["LIBRARY_PATH"] = "#{CREW_LIB_PREFIX}"`.  Otherwise, armv7l and i686 will not work.

Reply:
The `--build`, `--host`, `--target` flags in the configuration kill this too.  Maybe those aren't necessary to specify?  I can try without.